### PR TITLE
v1.9.0 update updates

### PR DESCRIPTION
**Pull request by StephenButtolph**

### CODE REVIEW

File: db/dbmodel.go [hunk 1]
```diff
@@ -2478,9 +2454,11 @@ func (p *persist) InsertCvmLogs(
 }
 
 type PvmProposer struct {
-	ID            string
-	ParentID      string
-	BlkID         string
+	ID       string
+	ParentID string
+	BlkID    string
+	// Deprecated: This is populated with the hash of the proposervm's block,
```
Comment (patrick-ogrady) [inline db/dbmodel.go:2460]: nit: replace `This` with what is actually deprecated.

File: services/indexes/pvm/writer.go [hunk 2]
```diff
@@ -403,54 +308,30 @@ func (w *Writer) indexBlock(ctx services.ConsumerCtx, proposerblockBytes []byte)
...
-			w.indexTransaction(ctx, blkID, *blk.Tx, false),
-		)
+		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeProposal, blk.CommonBlock, innerBlockBytes))
 	case *blocks.ApricotAbortBlock:
-		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeAbort, blk.CommonBlock, blockBytes))
+		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeAbort, blk.CommonBlock, innerBlockBytes))
 	case *blocks.ApricotCommitBlock:
-		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeCommit, blk.CommonBlock, blockBytes))
+		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeCommit, blk.CommonBlock, innerBlockBytes))
 	case *blocks.BlueberryProposalBlock:
```
Comment (patrick-ogrady) [inline services/indexes/pvm/writer.go:322]: Don't we need to update this to Banff?

File: services/indexes/pvm/writer.go [hunk 3]
```diff
@@ -209,52 +122,59 @@ type PtxDataModel struct {
...
-		w.initCtx(block)
-		blkID := ids.ID(hashing.ComputeHash256Array(b))
-		blkIDS := blkID.String()
-		btype := reflect.TypeOf(block)
+func (w *Writer) ParseJSON(b []byte) ([]byte, error) {
+	// Try and parse as a tx
+	tx, err := txs.Parse(blocks.GenesisCodec, b)
+	if err == nil {
+		tx.Unsigned.InitCtx(w.ctx)
+		txtype := reflect.TypeOf(tx) // Should this be `tx.Unsigned`?
```
Comment (patrick-ogrady) [inline services/indexes/pvm/writer.go:130]: I think that would mean the current endpoint just returns junk (or always `SignedTx` or something)?

Comment (StephenButtolph) [reply services/indexes/pvm/writer.go:130]: Yeah - I think that's correct

**Review by learyce — APPROVED** [review]

**Pull request merged**